app/controllers/taxons_controller.rb

```diff
@@ -31,6 +31,7 @@ def taxon_overview_and_child_taxons(taxon)
           'title' => current_taxon_title,
           'description' => '',
           'base_path' => current_taxon_title.downcase.tr(' ', '-'),
+          'is_current_taxon' => true,
```

Comment:
I would probably rename this to `has_tagged_content`, which is the trigger for having General Information and Guidance.